tracer/build/_build/Build.Steps.cs

```diff
@@ -1264,8 +1264,8 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                     (project, project.TryGetTargetFrameworks(), project.RequiresDockerDependency()) switch
                     {
                         _ when exclude.Contains(project.Path) => false,
-                        _ when !string.IsNullOrWhiteSpace(SampleName) => project.Path.ToString().Contains(SampleName, StringComparison.OrdinalIgnoreCase),
                         (_, _, DockerDependencyType.All) => false, // can't use docker on Windows
+                        (_, { } targets, _) when !string.IsNullOrWhiteSpace(SampleName) => project.Path.ToString().Contains(SampleName, StringComparison.OrdinalIgnoreCase) && targets.Contains(Framework),
```

Comment:
These were intentionally the other way around previously - the point being that we _want_ to ignore the docker dependency thing on Windows when you are running locally and provide a sample name 🤔

Reply:
Reverted in 9c61e57cfeaf14a61b9e24081d47931654d23fd6 . If we run into troubles with this again we can revisit 😄